virttest/libvirt_xml/checkpoint_xml.py

```diff
@@ -57,3 +58,20 @@ def marshal_to_disks(tag, attr_dict, index, libvirtxml):
         if tag != 'disk':
             return None
         return dict(attr_dict)
+
+    @staticmethod
+    def new_from_checkpoint_dumpxml(name, checkpoint_name, options="",
+                                    virsh_instance=base.virsh):
+        """
+        Return new CheckpointXML instance from virsh checkpoint-dumpxml cmd
+
+        :param name: vm name
+        :param checkpoint_name: checkpoint name
+        :param options: options passed to checkpoint-dumpxml
+        :param virsh_instance: virsh module or instance to use
+        :return: New initialized CheckpointXML instance
+        """
+        checkpoint_xml = CheckpointXML(virsh_instance=virsh_instance)
+        result = virsh_instance.checkpoint_dumpxml(name, checkpoint_name, options)
+        checkpoint_xml['xml'] = results_stdout_52lts(result).strip()
```

Comment:
Can we replace results_stdout_52lts(result) to result.stdout_text? There is a PR 2450 to remove 52lts compatiblity.

Reply:
thx for the reminder, updated